javatests/arcs/core/data/expression/EvaluatorParticleTest.kt

```diff
@@ -42,23 +41,19 @@ class EvaluatorParticleTest {
         //     value: x.value * scalar.magnitude
         //   }
         val scaledNumbersExpression = from("x") on seq("inputNumbers") select
-            new("Value")() {
-                listOf(
-                    "value" to scope("x").get<Scope, Number>("value") *
-                        scope("scalar")["magnitude"]
-                )
-            }
+            new("Value")(
```

Comment:
I really wish we could make this look like

new("Value") { ... } so it looks closer to the real syntax. I kind of have an idea how to do it, but it's more trouble than it's worth.